### PR TITLE
allow branding to specify more than one brand key

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

File: temba/middleware.py [hunk 1]
```diff
@@ -88,6 +87,10 @@ def get_branding_for_host(cls, host):
 
         if branding:
             branding["brand"] = brand_key
+
+            # set brands to just our brand key if nothing else is set
+            if "brands" not in branding:
+                branding["brands"] = [brand_key]
```
Comment (rowanseymour) [inline temba/middleware.py:93]: would it be clearer to call this something else... keys? aliases?

Comment (nicpottier) [reply temba/middleware.py:93]: Ya, I couldn't really decide on anything being better as opposed to just different. We are mapping to the `brand` field on an Org so `brands` felt most right in the end. Maybe `org_brands`?

Comment (nicpottier) [reply temba/middleware.py:93]: I guess maybe `aliases` could work? And we don't include the main one?

Comment (rowanseymour) [reply temba/middleware.py:93]: That seems the easiest to understand

**Pull request merged**